src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -65,56 +67,52 @@ public static class ArmConstants { // To find values later
 
 		// Feed Forward
 		public static final double ARM_K_S = 0.1;
-		public static final double ARM_K_V = 0;
-		public static final double ARM_K_A = 0;
-		public static final double ARM_K_G = 0.1;
+		public static final double ARM_K_V = 1.95;
+		public static final double ARM_K_A = 0.04;
+		public static final double ARM_K_G = 0.81;
 
 		public static final double WRIST_K_S = 0.1;
-		public static final double WRIST_K_V = 0;
-		public static final double WRIST_K_A = 0;
-		public static final double WRIST_K_G = 0.1;
+		public static final double WRIST_K_V = 1.95;
+		public static final double WRIST_K_A = 3.43;
+		public static final double WRIST_K_G = 0;
```

Comment:
Where did these values come from?

Reply:
filler values from recalc

Reply:
Can we revert them for now? The values from recalc probably won't work, whereas the values we had before were (sort of) working.